### PR TITLE
fix(Core/Scripts): ScriptEffect Frog Kiss

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [x] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6174

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

https://wowgaming.altervista.org/aowow/?spell=62536
https://www.wowhead.com/quest=13666/a-blade-fit-for-a-champion#comments

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

.quest add 13666 for alliance, .quest add 13673 for horde
.go xyz 3812.86 -4357.22 182.88 571

Use item then /kiss a frog, buff should disappear
When Maiden of Ashwood spawns, talk to her, she should have only 1 gossip option

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/game/Spells/SpellEffects.cpp [hunk 1]
```diff
@@ -3972,6 +3972,11 @@ void Spell::EffectScriptEffect(SpellEffIndex effIndex)
                             m_caster->CastSpell(m_caster, 61268, true);
                             return;
                         }
+                    case 62536: // Removes Warts-B-Gone Lip Balm
+                        {
+                            m_caster->RemoveAurasDueToSpell(62574);
+                        }
+                        break;
```
Comment (Nefertumm) [inline src/server/game/Spells/SpellEffects.cpp:3979]: Oh no, don't add more of those here, create a script for the spell instead

Comment (avarishd) [reply src/server/game/Spells/SpellEffects.cpp:3979]: But I like it there :(

File: data/sql/updates/pending_db_world/rev_1697002994350126800.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-- Duplicated gossip option
+DELETE FROM `gossip_menu_option` WHERE (`MenuID` = 10316) AND (`OptionID` = 1);
+
+DELETE FROM `spell_script_names` WHERE `spell_id`=62536;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1697002994350126800.sql:4]: spell_id and scriptname

File: src/server/game/Spells/SpellEffects.cpp (hunk 1 above)
Comment (Nefertumm) [reply src/server/game/Spells/SpellEffects.cpp:3979]: It's still here >:(

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**